### ISSUE TITLE
Confirm e-mail address changes

### ISSUE THREAD

**Issue by leofeyer** [contao/core-bundle#542]
@contao/developers I would like to add some kind of e-mail address confirmation routine to Contao.

If a member changes their e-mail address, the change shall only be applied after they have clicked a link in the confirmation e-mail. Thus I want to prevent that an invalid e-mail address is entered and there is no more way to contact the member.

What do you think about it?

**Comment by aschempp** [comment]
Good idea, but I wonder if it should be limited to user email or be extendable to basically anything?

**Comment by bytehead** [comment]
Is this really a core related feature?

**Comment by leofeyer** [comment]
Yes, I think so. We already require members to confirm their e-mail addresses when signing up or when subscribing to a newsletter, so why not require confirmation if an e-mail address changes, too? Would be consistent opt-in.

**Comment by leofeyer** [comment]
As discussed in Mumble on July 21st, there should be an opt-in service which can handle arbitrary tables and fields. The service can then be used by the personal data module and the newsletter bundle.

**Comment by leofeyer** [comment]
See contao/contao#196 for the opt-in service.
